jaxrs/src/test/java/com/cerner/beadledom/jaxrs/provider/FakeModel.java

```diff
@@ -28,6 +28,10 @@
   @JsonProperty("inner_models")
   public List<FakeInnerModel> innerModels;
 
+  public FakeModel() {
```

Comment:
why did you add a default constructor? the only place I see you using it is here where you use the constructor that already exist.